src/libs/blueprint/conduit_blueprint_mesh_utils.cpp

```diff
@@ -2066,23 +2066,18 @@ topology::unstructured::points(const Node &n,
     ntemp.set_external(n);
     generate_offsets(ntemp, ntemp["elements/offsets"]);
 
-    Node temp;
     const ShapeType topo_shape(ntemp);
 
     std::set<index_t> pidxs;
     if(!topo_shape.is_poly())
     {
         const Node &poffs_node = ntemp["elements/offsets"];
-        temp.set_external(DataType(poffs_node.dtype().id(), 1),
-            (void*)poffs_node.element_ptr(ei));
-        const index_t eoff = temp.to_index_t();
+        const index_t eoff = poffs_node[ei].value();
```

Comment:
This is where the issue is, we need the index_t_assessor, not try to ask the node for its child.